l5kit/l5kit/environment/reward.py

```diff
@@ -0,0 +1,141 @@
+from abc import ABC, abstractmethod
+from typing import DefaultDict, Dict, List, NamedTuple, Optional
+
+import numpy as np
+import torch
+
+from l5kit.environment.cle_metricset import L5BaseMetricSet, L5GymCLEMetricSet, SimulationOutputCLE
+from l5kit.simulation.dataset import SimulationDataset
+from l5kit.simulation.unroll import UnrollInputOutput
+
+
+class RewardInput(NamedTuple):
+    """ The input tuple to calculate reward
```

Comment:
```suggestion
    """The input tuple to calculate reward
```

Remove the space in the beginning of the docstrings. [Reference for example](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#an-example-class-with-docstrings).